playwright.config.ts

```diff
@@ -0,0 +1,81 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './src/e2e-tests',
```

Comment:
@Irev-Dev Do you think we could do something like `e2e/playwright`?

I put the tauri stuff at `e2e/tauri`, was thinking it made sense to (a) have them outside src* and (b) have both mechanisms in a single `e2e` top dir so we could get a quick understanding of coverage by looking at the list of spec files

Reply:
Yeah that's fair, I'll shuffle things.

Reply:
Thanks for thinking these things through 👏